repository/process.go

```diff
@@ -3,11 +3,27 @@ package repository
 import (
 	"github.com/google/go-github/v32/github"
 	"github.com/gruntwork-io/git-xargs/config"
+	"github.com/gruntwork-io/git-xargs/types"
 	"github.com/gruntwork-io/go-commons/logging"
 	"github.com/remeh/sizedwaitgroup"
 	"github.com/sirupsen/logrus"
 )
 
+func pullRequestWorker(gitxargsConfig *config.GitXargsConfig, pr types.OpenPrRequest, wg *sizedwaitgroup.SizedWaitGroup) {
```

Comment:
NIT: the name `pullRequestWorker` makes it sound like it fires up the worker itself, whereas this method is really doing the processing within a worker. Consider renaming to something like `openPullRequestsWithThrottling` or similar. 

NIT: add the method comment as suggested by @rhoboat and cross-link to the GitHub issue for more context.

Reply:
Thanks, done in 6c68714